frontend/public/components/utils/resource-log.jsx

```diff
@@ -45,6 +50,24 @@ const LogControls = ({dropdown, onDownload, toggleFullscreen, isFullscreen, stat
       {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
     </div>
     <div className="co-toolbar__group co-toolbar__group--right">
+      {!_.isEmpty(podLogLinks) && _.map(_.sortBy(podLogLinks, 'metadata.name'), (link, i) => {
+        const namespace = resource.metadata.namespace;
+        const namespaceFilter = link.spec.namespaceFilter;
+        // if namespaceFilter, do not output link if namespace does not match namespaceFilter regex
+        if (namespaceFilter && namespace.search(namespaceFilter) === -1) {
+          return null;
+        }
+        const url = link.spec.hrefTemplate
+          .replace('${resourceName}', encodeURIComponent(resource.metadata.name))
```

Comment:
Any chance someone might use `${resourceName}` or another parameter twice or more?

```javascript 
const url = link.spec.hrefTemplate
          .replace(/\$\{resourceName\}/g, encodeURIComponent(resource.metadata.name))
          .replace(/\$\{resourceUID\}/g', encodeURIComponent(resource.metadata.uid))
          .replace(/\$\{containerName\}/g, encodeURIComponent(containerName))
          .replace(/\$\{resourceNamespace\}/g, encodeURIComponent(namespace))
          .replace(/\$\{podLabels\}/g, JSON.stringify(resource.metadata.labels));
```  
I'm guessing it is not a stretch.

Reply:
> Any chance someone might use `${resourceName}` or another parameter twice or more?

This should be addressed now with the `g` flag from line 56